cudaq/jobs/docker/0.12/py3/requirements.txt

```diff
@@ -1,8 +1,8 @@
 # CUDA-Q quantum computing framework packages
-cudaq==0.12.0.post1
-cudaq-qec==0.4.0.post1
-cudaq-solvers==0.4.0
+cudaq==0.13.0
+cudaq-qec==0.5.0
+cudaq-solvers==0.5.0
+cupy-cuda13x==13.6.0
```

Comment:
why do we need to pin this? would the cudaq package above not bring in the right version?